### DISCUSSION TITLE
Bold Editor url shortening disable Android?

### DISCUSSION THREAD

**Discussion by wonderfullyrich** [standardnotes/forum#2714] [Suggestions]
I have StandardNotes running on two Macs, an iPhoneXS, and a Linage loaded Android phone.  I have a particular behavior in the Bold Editor that seems most apparent on my Android phone and is frustrating. Version 3.0.22 on my Android.

When I copy and paste a text only decently sized URL from a browser title bar, say for example: 
`https://web.archive.org/web/20200229174220/https://standardnotes.org/help/2/has-standard-notes-completed-a-third-party-security-audit `

My Android S. Notes will truncate the url to `https://web.archive.org/web/20...` and link to the full URL.  If I unlink the url, it retains the truncated form, not the full url.  I can select the link, edit it, manually re-add the full URL, and then unlink it to retain the full URL.  This seems to be the only platform that does this behavior.  All the other platforms copy and paste the URL in it's full length and in a linked form. 

Is there a way to disable the URL shortening feature and allow me to have a full unlinked URL without the manual process?

**Comment by JaspalSuri** [comment]
Hi @wonderfullyrich, 

Can you try making sure that all of your notes & tags are synced, sign out, and then sign back in to see if you can reproduce the issue? My guess is that the Bold Editor didn't update on that device for some strange reason.

**Comment by wonderfullyrich** [comment]
Same behavior after signout. Synced, signed out, closed and force closed the S.Note app, opened, signed in, synced, opened a note and pasted a long URL with the same truncated result.

-

-------- Original Message --------

**Comment by JaspalSuri** [comment]
Thanks for letting me know. I've made a report for it for the dev team to review. Currently the dev team is working on a several improvements under the hood, so it may take some time before they get around to fixing this particular feature.

**Comment by kennethshin1** [comment]
I faced the problem too. Try the site <a href="https://depositphotos.com/bgremover.html">remove bg</a>
